src/main/java/core/BasePage.java

```diff
@@ -20,7 +20,6 @@
     @FindBy(xpath = "//div[@id='logo']/a/img")
     protected WebElement homeRedirect;
 
-    //@FindBy(xpath = "//a[@href='http://184-dp.tk/index.php?route=information/contact']")
     @FindBy(xpath = "/html/body/footer/div/div/div[2]/ul/li[1]/a")
```

Comment:
**is there some not fullXpath?**

Reply:
Well, I did not find

Reply:
Maybe, it's better to use @FindBy(linkText="Contact Us")?